app/controllers/users_controller.rb

```diff
@@ -0,0 +1,81 @@
+class UsersController < ApplicationController
+  before_action :find_user, only: %i[ show edit update destroy user_profile update_profile change_password update_password ]
+
+  def index
+    @users = User.order(created_at: :desc)
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_param)
+    if @user.save
+      respond_to do |format|
+        flash[:notice] = "User is successfully created....👍"
+        format.js
+      end
+    else
+      respond_to do |format|
+        flash[:notice] = "something went wrong....👍"
+        format.js
+      end
+    end
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
```

Comment:
Don't needs to find user here as you already calling a before action to find a user.